randovania/games/prime_hunters/layout/prime_hunters_configuration.py

```diff
@@ -19,6 +19,7 @@ class HuntersOctolithConfig(BitPackDataclass, JsonDataclass):
 class HuntersConfiguration(BaseConfiguration):
     octoliths: HuntersOctolithConfig
     force_field_configuration: ForceFieldConfiguration
+    shuffled_force_fields: bool
```

Comment:
how come you have both `shuffled_force_fields` and `force_field_configuration`?

Reply:
I couldn't figure out how to make `on_preset_changed` work, so this was what I thought of.

Reply:
what wasn't working?

Reply:
I think it was something to do with the preset not showing the changes when reopening? I can't remember. If there's another way to compare if its been changed vs how I'm doing it now, then it'd probably be cleaner.